tests/components/knx/test_services.py

```diff
@@ -12,46 +12,53 @@ async def test_send(hass: HomeAssistant, knx: KNXTestKit):
     test_address = "1/2/3"
     await knx.setup_integration({})
 
-    # send DPT 1 telegram
-    await hass.services.async_call(
-        "knx", "send", {"address": test_address, "payload": True}, blocking=True
-    )
-    await knx.assert_write(test_address, True)
-
-    # send raw DPT 5 telegram
-    await hass.services.async_call(
-        "knx", "send", {"address": test_address, "payload": [99]}, blocking=True
-    )
-    await knx.assert_write(test_address, (99,))
-
-    # send "percent" DPT 5 telegram
-    await hass.services.async_call(
-        "knx",
-        "send",
-        {"address": test_address, "payload": 99, "type": "percent"},
-        blocking=True,
-    )
-    await knx.assert_write(test_address, (0xFC,))
-
-    # send "temperature" DPT 9 telegram
-    await hass.services.async_call(
-        "knx",
-        "send",
-        {"address": test_address, "payload": 21.0, "type": "temperature"},
-        blocking=True,
-    )
-    await knx.assert_write(test_address, (0x0C, 0x1A))
-
-    # send multiple telegrams
-    await hass.services.async_call(
-        "knx",
-        "send",
-        {"address": [test_address, "2/2/2", "3/3/3"], "payload": 99, "type": "percent"},
-        blocking=True,
-    )
-    await knx.assert_write(test_address, (0xFC,))
-    await knx.assert_write("2/2/2", (0xFC,))
-    await knx.assert_write("3/3/3", (0xFC,))
+    for response in (False, True):
```

Comment:
I'd use [pytest's paramterize](https://docs.pytest.org/en/6.2.x/parametrize.html) for the whole test in this case.
Also I think the tests will fail because the `response` attribute is not used in the `hass.services.async_call()` calls.